### DISCUSSION TITLE
Ordered lists need better and more flexible formatting

### DISCUSSION THREAD

**Discussion by jchatfranklin** [GitbookIO/community#763] [Feature Requests]
Greetings.  I've just gotten started with Gitbook, and I'm very impressed with how easy it is to just start using.  Bravo!

That said, of course, there is room for improvement.

First and foremost, and unless I'm missing something -- and I may be -- I see no way to modify the formatting of an indented numbered list.  That is, to avoid 1, 1.1, 1.1.1, and so on.  This is -- how shall I put this? -- awful, at least from a cognitive and instructional design perspective.  The first indented level must be lettered -- 1, a, b, c, and so on -- and we should have the option of upper case or lower case.  The succeeding level of indentation should be Roman numerals -- that is, 1, a, i, ii, iii, and so on.  And with all this you must be able to break the list with a non-ordered item, such as an image or a line of context, and continue on with the list -- thus, 1, a, b, image, c, d, i, image, ii, iii, and so on.

The attached file illustrates what I'm attempting to describe.

Finally, we also must be able to have a hybrid list -- 1, bullet, bullet, 2, bullet, letter, 3, and so on.

Speaking as a technical writer, this is not a trivial issue.  Lists are the bread and butter of good documentation, and were it up to me I'd consider this feature request as critical.

Please advise if this capability already exists and I have overlooked it.

Thank you very much.
[Proper ordered list example.docx](https://github.com/user-attachments/files/17420654/Proper.ordered.list.example.docx)

**Comment by denisemcg** [comment]
This is the same request https://github.com/orgs/GitbookIO/discussions/653

**Comment by JaniceManwiller** [comment]
In addition to the requested items - the sublist numbering has been driving me crazy forever - I'd also like to see more options to include a wider range of content in a list item.

I'm used to thinking of a list item as a container that includes one more or paragraphs, images, and even small tables.

I've been able to fake this with ending a list and then restarting it after adding an the image/table, but I shouldn't have to do that.
